GoogleSignIn/Tests/Unit/GIDGoogleUser+Testing.m

```diff
@@ -32,15 +33,14 @@ - (BOOL)isEqual:(id)object {
 - (BOOL)isEqualToGoogleUser:(GIDGoogleUser *)other {
   return [self.authentication isEqual:other.authentication] &&
       [self.userID isEqual:other.userID] &&
-      [self.serverAuthCode isEqual:other.serverAuthCode] &&
       [self.profile isEqual:other.profile] &&
-      [self.hostedDomain isEqual:other.hostedDomain];
+      [self.configuration isEqual:other.configuration];
```

Comment:
Do we have an `isEqual:` implementation for `GIDConfiguration`? If no, will this do what you want? I think it'll check for pointer equality, which may not be what is needed here.